docs/hooks/medication-prescribe.md

```diff
@@ -11,11 +11,15 @@ The set of medications proposed or in progress of being prescribed. All FHIR res
 Field | Priority | Prefetch Token | Description
 ----- | -------- | ---- | ----
 `medications` | REQUIRED | No | *array* DSTU2 - An array of MedicationOrder resources <br/> *array* STU3 - An array of MedicationRequest resources
+`patientId` | REQUIRED | No | *string*.  The FHIR `Patient.id` of the current patient in context
```

Comment:
I may have missed discussion here (sorry!) -- does this imply that the patient ID can't be used as an interpolated variable in pre-fetch expressions? (I'm assuming it can... but then I'm having trouble making sense of the "No" here.)

Reply:
I dont think you missed anything @jmandel. Thanks for the second set of eyes.

I need to go back and update these three hook definitions to include the top level string elements to describe them as being able to be used as an interpolated variable.

Reply:
Thanks for catching this @jmandel - I've fixed this on 0962406